### PR TITLE
Silence return type mismatch 

### PR DESCRIPTION
**Pull request by Jellyfrog**
...  To keep PHP7 compatibility

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/ObjectCache.php [hunk 1]
```diff
@@ -113,34 +114,30 @@ public function offsetGet($obj)
...
-    public function offsetSet($obj, $value)
+    public function offsetSet($obj, $value): void
     {
         if (! isset($this->data[$obj])) {
             $this->data[$obj] = [];
         }
 
         $this->data[$obj]['value'] = $value;
-
-        return $this->data[$obj]['value'];
```
Comment (Jellyfrog) [inline LibreNMS/ObjectCache.php:128]: It's tricky to find if this will break something

Comment (murrant) [reply LibreNMS/ObjectCache.php:128]: It should not

**Comment by murrant** [comment]
Did you test this on PHP7?

**Comment by Jellyfrog** [comment]
Test. What is this word you speak of? No i did not :)

**Comment by murrant** [comment]
Tested fine on PHP 7.3 :D

**Pull request merged**